tests/test_domain.py

```diff
@@ -166,3 +166,20 @@ def test_domain_fails_on_unknown_custom_slot_type(tmpdir):
             - utter_greet""")
     with pytest.raises(ValueError):
         TemplateDomain.load(domain_path)
+
+
+def test_domain_to_yaml():
+    test_yaml = """action_names:
+- utter_greet
+actions:
+- utter_greet
+config:
+  store_entities_as_slots: true
+entities: []
+intents: []
+slots: {}
+templates:
+  utter_greet:
+  - text: hey there!"""
+    domain = TemplateDomain.load_from_yaml(test_yaml)
+    assert test_yaml in domain.to_yaml()
```

Comment:
why 'in' ? would `x.strip() == y.strip()` also work?

Reply:
yep, that works fine!